libs/aws/odc/aws/queues.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+
+def get_queue(queue_name):
+    sqs = boto3.resource('sqs')
+    queue = sqs.get_queue_by_name(QueueName=queue_name)
+    return queue
+
+def publish_message(queue, msg):
+    resp = queue.send_message(
+        QueueUrl=queue.url,
+        MessageBody=msg
+    )
+
+    assert resp['ResponseMetadata']['HTTPStatusCode'] == 200, "Failed to publish the message"
+
+def get_messages(queue, limit):
+    count = 0
+    while True:
+        messages = queue.receive_messages(
+            VisibilityTimeout=60,
```

Comment:
What is `VisibilityTimeout=`. Is it not something we want to expose as a configurable parameter?
Same goes for other timeout.